src/core/service.c

```diff
@@ -2796,7 +2811,7 @@ static int service_start(Unit *u) {
          * please! */
         if (IN_SET(s->state,
                    SERVICE_STOP, SERVICE_STOP_WATCHDOG, SERVICE_STOP_SIGTERM, SERVICE_STOP_SIGKILL, SERVICE_STOP_POST,
-                   SERVICE_FINAL_WATCHDOG, SERVICE_FINAL_SIGTERM, SERVICE_FINAL_SIGKILL, SERVICE_CLEANING))
+                   SERVICE_FINAL_WATCHDOG, SERVICE_FINAL_SIGTERM, SERVICE_FINAL_SIGKILL, SERVICE_CLEANING, SERVICE_MOUNTING))
```

Comment:
This is unnecessary, as unit_start() should make this noop early if already active

Reply:
it won't hurt either, better leave it just in case since everything else is listed too

Reply:
Please remove it. We want to catch programming errors specifically through the assertion below. It would be spurious to put the job back to run queue by -EAGAIN.